src/Components/CreateImageWizardV2/steps/Review/ReviewStepTables.tsx

```diff
@@ -64,7 +66,15 @@ export const FSReviewTable = () => {
               <Th>Minimum size</Th>
             </Tr>
           </Thead>
-          <Tbody data-testid="file-system-configuration-tbody-review"></Tbody>
+          <Tbody data-testid="file-system-configuration-tbody-review">
+            {partitions.map((partition, partitionIndex) => (
+              <Tr key={partitionIndex}>
+                <Td className="pf-m-width-30">{partition.mountpoint}</Td>
+                <Td className="pf-m-width-30">xfs</Td>
+                <Td className="pf-m-width-30">{partition.min_size} </Td>
```

Comment:
What we want to display in this column is a specific size of the partition and the unit. Here's how the table looks like in V1:

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/9db54f31-b7f1-45ee-83a7-87c1f7aadc02)

Reply:
There are probably two ways to handle this - either we store both the value and the unit and then use those or we could save the size in bytes and do a small computation before displaying the minimum size including the unit. How did we work with the values in V1?

Reply:
Oops, I didn't realize we would also need to remember the display units for the review step.

In that case, let's add a 'displayUnits' field to the partition object we store in the wizardSlice's array of partitions. We will always store the min_size in bytes (I'm hoping that leads to less bugs when creating requests or editing the blueprint).